x/concentrated-liquidity/lp.go

```diff
@@ -91,6 +106,26 @@ func (k Keeper) withdrawPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAd
 		return sdk.Int{}, sdk.Int{}, err
 	}
 
+	pool, err := k.getPoolById(ctx, poolId)
+	if err != nil {
+		return sdk.Int{}, sdk.Int{}, err
+	}
+
+	denom0 := pool.GetToken0()
+	denom1 := pool.GetToken1()
+
+	// send withdraw amount from pool to position owner
+	if actualAmount0.IsPositive() && actualAmount1.IsPositive() {
+		err = k.bankKeeper.SendCoins(ctx, pool.GetAddress(), owner, sdk.NewCoins(sdk.NewCoin(denom0, actualAmount0), sdk.NewCoin(denom1, actualAmount1)))
+	} else if actualAmount0.IsPositive() && actualAmount1.IsZero() {
+		err = k.bankKeeper.SendCoins(ctx, pool.GetAddress(), owner, sdk.NewCoins(sdk.NewCoin(denom0, actualAmount0)))
+	} else if actualAmount0.IsZero() && actualAmount1.IsPositive() {
+		err = k.bankKeeper.SendCoins(ctx, pool.GetAddress(), owner, sdk.NewCoins(sdk.NewCoin(denom1, actualAmount0)))
+	}
+	if err != nil {
+		return sdk.Int{}, sdk.Int{}, err
```

Comment:
Note to reviewer: does a normal send coins work here or do we need to eventually change this to a module account send

Reply:
`SendCoinsFromModuleToAccount`

Reply:
I actually dont think we use FromModuleToAccount here. I believe in gamm we use SendCoins. Plus SendCoinsFromModuleToAccount only takes a module name, which we wouldn't be able to do for each pool